### DISCUSSION TITLE
Numberinput props editable make attribute readonly ineffective.

### DISCUSSION THREAD

**Discussion by elmatou** [buefy/buefy#3998] [Ideas]
Hi,

Numberinput component is an input field with decorations and handlers.

There is a prop called `editable` which is basically the opposite  to `readonly` and breaks standard HTML attribute.

https://github.com/buefy/buefy/blob/1cb7ba3937949e1a9d2246c3784fd73b8e8d169b/src/components/numberinput/Numberinput.vue#L120-L123

as the attributes are passed to the input (via b-input)
https://github.com/buefy/buefy/blob/1cb7ba3937949e1a9d2246c3784fd73b8e8d169b/src/components/numberinput/Numberinput.vue#L39

 the `editable` prop is not useful and breaks basic assumptions.
https://github.com/buefy/buefy/blob/1cb7ba3937949e1a9d2246c3784fd73b8e8d169b/src/components/numberinput/Numberinput.vue#L45

I propose to remove it in further major release.

**Comment by fabiofdsantos** [comment]
Does it makes sense to replace `:readonly="!editable"` with `:disabled="!editable"`?

ping @jtommy @service-paradis

**Comment by jtommy** [comment]
Probably yes, but it's a breaking change thai I wouldn't add as described https://github.com/buefy/buefy/issues/2505#issuecomment-917645589

**Comment by fabiofdsantos** [comment]
Hmm I was looking for a way to change the css of the input element to "disabled" instead of "read-only" when editable is "false". 

I don't think it's a breaking change.. Just to understand if the framework would handle it or if we need to apply some custom css 🙂
